e2e/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this an empty file?

Reply:
I think it is needed by cargo so its a crate. All the e2e tests seem to be implemented as integration tests. I might be wrong though.

Reply:
No, that's it